source/includes/azure/_disks.md

```diff
@@ -245,3 +245,53 @@ curl -X POST \
 <code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/disks/:id?operation=detach</code>
 
 Detach an existing disk from a given [instance](#azure-instances).
+
+<!-------------------- ATTACH A DISK -------------------->
+
+#### Attach a disk
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/azure-conn/test_env/disks/subscriptions/60a3c9eebe64-55c1-4b1d-969b-60a3c/resourceGroups/azure-connect-system-ssamadh-mean-env/providers/Microsoft.Compute/disks/the-sheep_disk1_c0cbe27c49aa928525e93bd2519aac4?operation=attach"
+
+# Request should look like this
+```
+```shell
+# Request Example:
+```
+
+```json
+{
+  "instanceId" : "/subscriptions/60a3c9eebe64-55c1-4b1d-969b-60a3c/resourceGroups/azure-connect-system-ssamadh-mean-env/providers/Microsoft.Compute/disks/cool-instance",
+}
+```
+```shell
+# Request Example 2:
+```
+
+```json
+{
+  "instanceId" : "/subscriptions/60a3c9eebe64-55c1-4b1d-969b-60a3c/resourceGroups/azure-connect-system-ssamadh-mean-env/providers/Microsoft.Compute/disks/cool-instance",
+  "lun" : 10
+}
+```
+
+```shell
+# Response Example
+```
+```json
+{
+  "taskId": "802c5ae0-8431-47dc-9abe-7e10a9badddc",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/disks/:id?operation=attach</code>
+
+Attach a disk to an instance
+
+Optional | &nbsp;
+---------- | -----
+LUN: `lun`<br/>*Integer* | The logic unit number of a disk. Can only be between 1 and 63 inclusive. If the LUN is not specified it will default to the first available LUN for that instance.
```

Comment:
No need for the `LUN: `

Reply:
ab36c1c83d0980c67fd84a0f0cafeaace182f146 
removed it